### PR TITLE
updaete override-policy.md

### LINKED ISSUE #291: Redundant fieldSelector section in Override Policy document
**Task description**:

We have a duplicate section in `Override Policy`'s introduction, you can help clean up

image

**Solution**:

Update the following file:

```
docs\userguide\scheduling\override-policy.md
i18n\zh\docusaurus-plugin-content-docs\current\userguide\scheduling\override-policy.md
versioned_docs\version-v1.4.0\userguide\scheduling\override-policy.md
i18n\zh\docusaurus-plugin-content-docs\version-v1.4.0\userguide\scheduling\override-policy.md
```

About how to contribute, you can refer to: https://karmada.io/zh/docs/contributor/contribute-docs/

**Who can join or take the task**:

The good first issue is intended for `first-time contributors` to get started on his/her contributor journey.

After a contributor has successfully completed 1-2 good first issue's,
they should be ready to move on to `help wanted` items, saving the remaining `good first issue` for other new contributors.

**How to join or take the task**:

Just reply on the issue with the message `/assign` in a separate line.

Then, the issue will be assigned to you.

**How to ask for help**:

If you need help or have questions, please feel free to ask on this issue.
The issue author or other members of the community will guide you through the contribution process.

### PR DESCRIPTION
**Pull request by fengshunli**
Signed-off-by: fsl <1171313930@qq.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #291

**Special notes for your reviewer**:

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm
/cc @Poor12 @RainbowMango

**Comment by fengshunli** [comment]
> Hi @fengshunli, it seems that a 1.3 zh file is missing.

zh I did not change @XiShanYongYe-Chang

**Comment by fengshunli** [comment]
I found that there is still a problem with the original document here 
image

File: versioned_docs/version-v1.3.0/userguide/scheduling/override-policy.md [hunk 1]
```diff
@@ -91,29 +91,6 @@ spec:
...
-          matchExpressions:
-            - key: region
-              operator: In
-              values:
-                - cn-north-1
-      overriders:
-      ...
-```
-It means override rules above will only be applied to those resources propagated to clusters which has the `spec.region` field with values in [cn-north-1].
-
```
Comment (RainbowMango) [inline versioned_docs/version-v1.3.0/userguide/scheduling/override-policy.md:116]: Nice finding!
The version-1.3.0 contains the redundant thing, thanks for doing this.
But could you please also remove it from the zh one:
https://github.com/karmada-io/website/blob/main/i18n/zh/docusaurus-plugin-content-docs/version-v1.3.0/userguide/scheduling/override-policy.md#fieldselector-1

**Comment by XiShanYongYe-Chang** [comment]
> I found that there is still a problem with the original document here

可以再稍微提醒下我有什么问题么，我没有看出来

**Comment by RainbowMango** [comment]
> I found that there is still a problem with the original document here

@Poor12 
Please take a look. I think a document shouldn't be present if it only has the Chinese version. It's a bad experience for English readers.

File: versioned_docs/version-v1.3.0/userguide/scheduling/override-policy.md (hunk 1 above)
Comment (fengshunli) [reply versioned_docs/version-v1.3.0/userguide/scheduling/override-policy.md:116]: update

**Comment by fengshunli** [comment]
updated @XiShanYongYe-Chang @RainbowMango

**Comment by fengshunli** [comment]
> > I found that there is still a problem with the original document here
> 
> @Poor12 Please take a look. I think a document shouldn't be present if it only has the Chinese version. It's a bad experience for English readers.

image
what does this translate into, give a title, I will change it @XiShanYongYe-Chang @RainbowMango

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve
Thanks, and welcome to Karmada~

**Pull request merged**